packages/zql/src/mutate/mutator.ts

```diff
@@ -223,6 +235,11 @@ export type Mutator<
     ctx: TContext;
     tx: Transaction<TSchema, TWrappedTransaction>;
   }) => Promise<void>;
+
+  /**
+   * Type-only phantom property to surface mutator types in a covariant position.
+   */
+  ['~']: Expand<MutatorTypes<TInput, TSchema, TContext, TWrappedTransaction>>;
```

Comment:
Calling this out - added this since we'll have trouble testing schema types on the return types of `defineMutators` unless we have this. We already have something similar for `CustomQuery`.